### ISSUE TITLE
Comment Moderation feature review

### ISSUE THREAD

**Issue by cbunicef** [unicef/iogt#1419]
### Needed before for MVP approval

- [ ] Table must be responsive - recommend on narrower screens we have the comment stacked above the buttons - @sheralim012 I discovered this today, this needs to be done before invoicing
- [ ] Change footer title to "Comment Moderation"
- [ ] Confirm that the old Hidden/Shown in Admin Panel is the same as the new Published/Unpublished
- [ ] Are comments that were Removed in the previous Admin Panel interface shown in the new Admin Panel interface?

### Changes needed after MVP

- [ ] HTML title for the Comment Moderation page
- [ ] Date information for each comment
- [ ] For filter views EXCEPT Unmoderated, only the last 24 hours should be shown
- [ ] Escalating permissions, so that the AI moderation can be superseded by community moderated, which is superseded by admin moderation

**Comment by sheralim012** [comment]
> * [ ]  Confirm that the old Hidden/Shown in Admin Panel is the same as the new Published/Unpublished

- No, Hidden/Shown is related to is_removed and was removed from the admin panel so the user wouldn't get confused about it and Published/Unpublished
- Published/Unpublished is related to is_public

> * [ ]  Are comments that were Removed in the previous Admin Panel interface shown in the new Admin Panel interface?

- all comments posted after the deployment of this feature will be visible in the community moderation panel, but admin panel still list down all the comment previous and new

**Comment by sheralim012** [comment]
> * [ ]  Table must be responsive - recommend on narrower screens we have the comment stacked above the buttons - @sheralim012 I discovered this today, this needs to be done before invoicing

could you review this https://www.loom.com/share/9f7e98c98cf4413ea38a8630f657fa3a
